solidity/test/support/helpers.js

```diff
@@ -1,18 +1,57 @@
+const clWallet = require('../../app/cl_wallet.js');
+const clUtils = require('../../app/cl_utils.js');
+deploy = require('../../app/deploy.js');
```

Comment:
Are we missing a `const` here?

Reply:
That deploy function is made global as a convenience for later test setup. We could instead attach all of our test helpers to one object that we import at the top of every test file.

Reply:
I think test helper should define its own global `deploy` function that and sets things like `key = 'c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3'`. I would prefer to pass in a constructed wallet and provider than using the global state as we are now. That could be for another PR.

Reply:
Agreed, I think we could reduce a lot of global state with dependency injection. There's a lot of changes involved there, let's circle back on that in a subsequent PR.